doc/guide/https.xml

```diff
@@ -61,7 +64,6 @@ $ sudo /usr/lib/cockpit/cockpit-certificate-ensure --check
 getcert request -f /etc/cockpit/ws-certs.d/50-certmonger.cert \
                 -k /etc/cockpit/ws-certs.d/50-certmonger.key \
                 -D myhostname.example.com \
-                -m 640 -o root:cockpit-ws -M 644 \
```

Comment:
Let's do the same change in src/tls/cockpit-certificate-helper.in and test/verify/check-connection then, to validate this.

Reply:
See the commit message for why I didn' do it in certificate-helper: cockpit-ws still needs the correct permissions.

Changing the integration test is absolutely a good idea, though.